test/integration/acceptance/basic_mst_state_propagation.cpp

```diff
@@ -82,8 +82,9 @@ class BasicMstPropagationFixture : public AcceptanceFixture {
  * @when such transaction is sent to one of two iroha peers in the network
  * @then that peer propagates MST state to another peer
  */
+// TODO: 2019-03-27 @muratovv the test is blocked by IR-276
 TEST_F(BasicMstPropagationFixture,
-       MstStateOfTransactionWithoutAllSignaturesPropagtesToOtherPeer) {
+       DISABLED_MstStateOfTransactionWithoutAllSignaturesPropagtesToOtherPeer) {
```

Comment:
Why is the test disabled? Please provide a reason. Taking into account another failing test related to MST (see CI), I suggest possible investigating MST regression - it could cause this test failing.

Reply:
Seems, the branch was outdated and now basic_mst_state_propagation test absent.